src/mui/button/CancelButton.js

```diff
@@ -16,11 +16,9 @@ const CancelButton = ({
         onClick={event => {
             event.stopPropagation();
         }}
-        containerElement={
-            <Link to={`${basePath}`} />
-        }
+        containerElement={<Link to={`${basePath}`} />}
```

Comment:
For next time: That's weird to have so much linter correction with that commit message